src/main/java/com/smartbear/collaborator/issue/IssueRest.java

```diff
@@ -739,10 +762,10 @@ public String createReview(CollabUserInfo collabUserInfo) throws Exception {
 			createReviewCommand.getArgs().put("title", issue.getSummary() != null ? issue.getSummary() : "");
 			createReviewCommand.getArgs().put("customFields",
 					new com.smartbear.collaborator.json.collab.CustomField[] { 
-					new com.smartbear.collaborator.json.collab.CustomField("Overview", new HashSet(Arrays.asList(issue.getDescription() != null ? issue.getDescription() : "")) )	});
-			createReviewCommand.getArgs().put("internalCustomFields",
-					new com.smartbear.collaborator.json.collab.CustomField[] { 
-					new com.smartbear.collaborator.json.collab.CustomField("JiraIssueId",  new HashSet(Arrays.asList(issue.getId().toString())) )});
+							new com.smartbear.collaborator.json.collab.CustomField("Overview", new HashSet(Arrays.asList(issue.getDescription() != null ? issue.getDescription() : "")) ),

```

Comment:
Oh okay, @ploko. That's good to know. How can I access that field and/or make it visible? I found that this code failed in my testing on our test Collaborator server, which is why I made the change. I also didn't see the "internalCustomFields" listed as valid in the API (though I might not have found where to look).

Would it be sufficient just to revert this change?